src/components/input/__tests__/input_text.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Our tests are, at this moment, using the extension `.test.tsx` instead of `.spec.tsx`. Shall we keep using the same extension?

Reply:
Possibly, although we could start using .spec.* instead of .test.* to indicate that a test is a newer test.

Reply:
I prefer .test.tsx.

Reply:
What is wrong with just `.test.tsx`?

Reply:
Nothing, it's just a name. I'll change it to .test.tsx when reviews are done.